src/Hide.js

```diff
@@ -3,23 +3,26 @@ import PropTypes from 'prop-types'
 import Box from './Box'
 import theme from './theme'
 
-const getMaxWidth = em => em - 0.01
-
-const breakpoints = props => ({
-  xs: `@media screen and (max-width: ${getMaxWidth(
-    props.theme.breakpoints[0]
-  )}em)`,
-  sm: `@media screen and (min-width: ${
-    props.theme.breakpoints[0]
-  }em) and (max-width: ${getMaxWidth(props.theme.breakpoints[1])}em)`,
-  md: `@media screen and (min-width: ${
-    props.theme.breakpoints[1]
-  }em) and (max-width: ${getMaxWidth(props.theme.breakpoints[2])}em)`,
-  lg: `@media screen and (min-width: ${
-    props.theme.breakpoints[2]
-  }em) and (max-width: ${getMaxWidth(props.theme.breakpoints[3])}em)`,
-  xl: `@media screen and (min-width: ${props.theme.breakpoints[3]}em)`
-})
+const getMaxWidth = width => width - 0.01
```

Comment:
The Design System WG meeting ran out of time when this topic was discussed.

To recap, the suggestion was to design this `width` prop to be flexible to support features similar to [this "width (responsive)" API in `styled-system`](https://github.com/jxnblk/styled-system/blob/master/docs/api.md#width-responsive).

My proposal is to omit the calculation from width units to em units to keep this PR simplistic, and use another open issue to discuss the robust, agreed way(s) to build similar "_responsive width_" support mentioned above, and then an engineer will implement the "_responsive width_" in `Hide` with agreed approach.